packages/react-component-library/src/components/RangeSlider/Handle.tsx

```diff
@@ -14,6 +17,97 @@ interface HandleProps {
   thresholds?: number[]
 }
 
+interface StyledHandleProps {
+  $isActive?: boolean
+  thresholdColor?: ThresholdColor
```

Comment:
I am just trying to develop and understanding of how we use transient props on styled-components. Would it make more sense if we default to use transient props for all styled-components and then only make it non transient (normal) if it needs to be drilled down? If this is the case then it would imply `thresholdColor` would be transient `$transientColor` because it does not need drilling down to `div`.

Reply:
Yeah ok that sounds like a sensible convention. 

So everything unique to the StyledComponent that doesn't need to be drilled to the wrapped component should be marked as transient.

Reply:
It feels like that makes sense rather than having special cases for transient props. Was trying to find other documentation on a convention like this.